lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
-      return true unless config.managed?
-
-      out = exec('status').read
-      out =~ /running on port #{port}/
+      exec('status').read
     rescue
-      false
+      'No status information available'
+    end
+
+    ##
+    # Is Solr running?
+    def started?
+      status =~ /running on port #{port}/ && true || false
```

Comment:
Personal preference, I like to see actual true or false instead of truthy or falsy values and this satisfies that craving.